packages/glimmer-runtime/lib/dom/change-lists.ts

```diff
@@ -126,6 +134,27 @@ export const InputValuePropertyChangeList = {
   }
 };
 
+export const TextAreaValueChangeList = {
+  setAttribute(dom: DOMHelper, element: Element, attr: string, value: Opaque) {
+    let textArea = <HTMLTextAreaElement>element;
+    let normalizedValue = normalizeTextValue(value);
+
+    if(textArea.childNodes.length === 0) {
+      let textNode = dom.createTextNode(normalizedValue);
+      textArea.appendChild(textNode);
+    } else {
+      let currentValue = textArea.value;
```

Comment:
It's a good point, the [initial issue](https://github.com/emberjs/ember.js/issues/14001) was to do with `{{textarea value=foo}}` where `foo=''`, not `<textarea value={{foo}} />`. 

Somewhere along the line I changed it to `<textarea>`, probably when trying to get a failing test in glimmer.